doc/gtk-3.0-gnucash.css

```diff
@@ -0,0 +1,86 @@
+/* Note: Widgets obtained from Glade files will not be addressable
```

Comment:
Perhaps we should rearrange this comment a bit to make it easier to follow for the users:
1. start by explaining this is a sample css file
2. then where it should be installed and how it should be named to actually use it (you can explain the paths on the 3 platforms here as well)
3. Then follow up with the note about the limitation on Glade file widgets